cookbook/templates/default/agent.yml.erb

```diff
@@ -0,0 +1,63 @@
+server:
+  type: simple
+  applicationContextPath: /baragon-agent/v2
+  connector:
+    type: http
+    port: 8882
```

Comment:
This should be configurable like the port for the service is, maybe with 8882 as the default